### PR TITLE
drop boost lexical_cast

### PR DESCRIPTION
**Pull request by FernetMenta**
another step to kill boost

### CODE REVIEW

**Comment by FernetMenta** [comment]
jenkins build this please

File: xbmc/ContextMenuItem.cpp [hunk 1]
```diff
@@ -36,7 +34,10 @@ std::string CContextMenuItem::GetLabel() const
     return "";
 
   if (StringUtils::IsNaturalNumber(m_label))
-    return m_addon->GetString(boost::lexical_cast<int>(m_label.c_str()));
+  {
+    int number = std::stoi(m_label);
```
Comment (Jalle19) [inline xbmc/ContextMenuItem.cpp]: I don't think this is available on Android. Jenkins will probably reveal whether I'm correct or not.

**Comment by FernetMenta** [comment]
jenkins build this please

**Comment by tamland** [comment]
already fixed this in #8819. no need

**Pull request closed without merging**